lib/openc_bot/company_fetcher_bot.rb

```diff
@@ -38,7 +38,7 @@ def primary_key_name
 
     def report_run_results(results)
       send_run_report(results)
-      report_run_to_oc(results)
+      report_run_to_analysis_app(results)
```

Comment:
Because it's doing something different. Leaving the old name would be confusing. We're not reporting to oc anymore :-) 

But I understand why you're asking now. We discussed how incorporating these changes within all our our bots is going to be ...quite a pain. And the bots call this method (to my surprise).  So the old name `report_run_to_oc` lives on as an alias for the time being. One less complication.